src/Easy.zig

```diff
@@ -127,28 +135,12 @@ pub const Response = struct {
 
             if (self.name) |filter_name| {
                 if (self.c_header) |c_header| {
-                    // Stop early.
+                    // fast path
                     if (c_header.*.index + 1 == c_header.*.amount) {
                         return null;
                     }
                 } else {
-                    // `curl_easy_header` is slightly more performant for the first header.
-                    if (errors.headerErrorFrom(c.curl_easy_header(
-                        self.handle,
-                        filter_name.ptr,
-                        0,
-                        c.CURLH_HEADER,
-                        request,
-                        &self.c_header,
-                    ))) |err| return switch (err) {
-                        error.Missing, error.NoHeaders => null,
-                        else => err,
-                    };
-                    const c_header = self.c_header orelse unreachable;
-                    return Header{
-                        .c_header = c_header,
-                        .name = std.mem.sliceTo(c_header.*.name, 0),
-                    };
+                    return Response.getHeaderInner(self.handle, filter_name, &self.c_header);
```

Comment:
You missed `request` here.

Reply:
Fixed in https://github.com/jiacai2050/zig-curl/commit/c18d706108913c40b307c95a2be27bd45fc0e52f